client/flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,7 @@
 		97C146FC1CF9000F007C117D /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FA1CF9000F007C117D /* Main.storyboard */; };
 		97C146FE1CF9000F007C117D /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FD1CF9000F007C117D /* Assets.xcassets */; };
 		97C147011CF9000F007C117D /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FF1CF9000F007C117D /* LaunchScreen.storyboard */; };
+		C2308E162432A138003B2CD8 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = C2308E152432A137003B2CD8 /* GoogleService-Info.plist */; };
```

Comment:
isn't there already a GoogleService-Info.plist checked into the repo? is this Xcode being weird / messing up the linking?

Reply:
No, the developer is being weird and messed up the linking :) Fixed.